### PR TITLE
Add selected text Web search functionality

### LINKED ISSUE #2363: Web search for highlighted keyword
https://support.andbible.org/scp/tickets.php?id=1671

> would it be possible to add a web search shortcut to a highlighted keyword? that would be helpful!

### PR DESCRIPTION
**Pull request by wesinator**
**Describe the pull request content**
Adds a web search selection menu option
using `app.SearchManager`

closes #2363

Disclaimer: I am not an Android dev, I may have missed something.

[NO_TRAIN]::

### CODE REVIEW

**Review by tuomas2 — APPROVED** [review]
Looks good to me

**Comment by tuomas2** [comment]
I just want this to go directly to current-stable, so changing target branch

File: app/src/main/res/menu/bibleview_selection.xml [hunk 1]
```diff
@@ -53,6 +53,12 @@
 			android:visible="false"
 			app:showAsAction="always"
 			/>
+		<item android:id="@+id/web_search"
+			android:title="@string/web_search"
+			android:orderInCategory="2"
+			app:showAsAction="always"
+			android:visible="false"
```
Comment (tuomas2) [inline app/src/main/res/menu/bibleview_selection.xml:60]: Hmm, looks like this is always hidden. did you try this at all yourself?

Comment (wesinator) [reply app/src/main/res/menu/bibleview_selection.xml:60]: sorry - I didn't get a chance to build/test

I thought this meant it was only visible in the overflow section of the selection menu - does it not dispkay the web search at all??

Thanks for merging these!

**Pull request merged**